### PR TITLE
Update ASUS.xml

### PR DESCRIPTION
**Pull request by koops76**

### CODE REVIEW

File: src/chrome/content/rules/ASUS.com.xml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	<target host="vip.asus.com" />
+
+	<securecookie host=".+" name=".+" />
+
+	<rule from="^http://asus\.com/" to="https://www.asus.com/" />
+
+	<rule from="^http://shop\.asus\.com/" to="https://eshop.asus.com/" />
+
+	<rule from="^http:" to="https:" />
+</ruleset>
```
Comment (J0WI) [inline src/chrome/content/rules/ASUS.com.xml]:
```
WFM:
advantage.asus.com
apps.asus.com
asuscontrolcenter.asus.com
csr.asus.com
ca.estore.asus.com
demeter.asus.com
dlcdnimgs.asus.com
dlsvr04.asus.com
zenbo.asus.com
zenui.asus.com
edgeup.asus.com
event.asus.com
etrk.asus.com
qr.asus.com
press.asus.com
talent.asus.com
techinstyle.asus.com

non-working:
commercialsupport.asus.com
www.commercialsupport.asus.com
myzen.asus.com (r)
sonicmaster.asus.com (404)
store.asus.com (MCB)
emditpison.asus.com (m)
dlcdnet.asus.com (m)
promos.asus.com (t)

File: src/chrome/content/rules/ASUS.com.xml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+	<target host="rog.asus.com" />
+	<target host="serviceshop.asus.com" />
+	<target host="shop.asus.com" />
+	<target host="vip.asus.com" />
+
+	<securecookie host=".+" name=".+" />
+
+	<rule from="^http://asus\.com/" to="https://www.asus.com/" />
+
+	<rule from="^http://shop\.asus\.com/" to="https://eshop.asus.com/" />
```
Comment (J0WI) [inline src/chrome/content/rules/ASUS.com.xml]: Some content is different

**Comment by koops76** [comment]
@J0WI Done.

File: src/chrome/content/rules/ASUS.com.xml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+	<target host="eshop.asus.com" />
+	<target host="etrk.asus.com" />
+	<target host="event.asus.com" />
+	<target host="mymail.asus.com" />
+	<target host="pcdiy.asus.com" />
+	<target host="press.asus.com" />
+	<target host="qr.asus.com" />
+	<target host="rog.asus.com" />
+	<target host="serviceshop.asus.com" />
+	<target host="shop.asus.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/ASUS.com.xml]: Move this to the comments.

Comment (koops76) [inline src/chrome/content/rules/ASUS.com.xml]: @J0WI Done.

**Pull request merged**